nifi-registry/nifi-registry-core/nifi-registry-properties/src/main/java/org/apache/nifi/registry/properties/NiFiRegistryProperties.java

```diff
@@ -479,4 +482,23 @@ public String getOidcClaimIdentifyingUser() {
         return getProperty(SECURITY_USER_OIDC_CLAIM_IDENTIFYING_USER, "email").trim();
     }
 
+    /**
+     * Returns the network interface list to use for HTTPS. This method returns a mapping of
+     * network interface property names to network interface names.
+     *
+     * @return the network interface names of all HTTPS network interfaces
+     */
+    public Set<String> getHttpsNetworkInterfaceNames() {
+        final Set<String> networkInterfaceNames = new HashSet<>();
+
+        // go through each property
+        for (String propertyName : getPropertyKeys()) {
+            // determine if the property is a network interface name
+            if (StringUtils.startsWith(propertyName, WEB_HTTPS_NETWORK_INTERFACE_PREFIX)) {
+                // get the network interface property value
+                networkInterfaceNames.add(getProperty(propertyName));
```

Comment:
This can add an empty value to `networkInterfaceNames`, so this should be adjusted to filter out empty values.